tests/pipelines/pipeline_params.py

```diff
@@ -127,3 +127,5 @@
 TEXT_TO_IMAGE_CALLBACK_CFG_PARAMS = frozenset(["prompt_embeds"])
 
 VIDEO_TO_VIDEO_BATCH_PARAMS = frozenset(["prompt", "negative_prompt", "video"])
+
+IP_ADAPTER_PARAMS = frozenset(["ip_adapter_image"])
```

Comment:
I think we don't need this for now - so far all pipelines use `ip_adapter_image`